packages/synapse-react-client/src/components/Forum/ForumThreadEditor.stories.tsx

```diff
@@ -1,15 +1,15 @@
 import { Meta, StoryObj } from '@storybook/react'
-import { ForumThreadEditor } from './ForumThreadEditor'
 import { fn } from '@storybook/test'
+import { ForumThreadEditor } from './ForumThreadEditor'
 
 const meta = {
-  title: 'Markdown/MarkdownEditor',
+  title: 'Synapse/ForumThreadEditor',
   component: ForumThreadEditor,
```

Comment:
Create separate stories for ForumThreadEditor and MarkdownEditor to clarify each component's behavior